git-extra/vimrc

```diff
@@ -19,17 +19,16 @@ set statusline=%<%F%h%m%r\ [%{&ff}]\ (%{strftime(\"%H:%M\ %d/%m/%Y\",getftime(ex
 "------------------------------------------------------------------------------
 " Only do this part when compiled with support for autocommands.
 if has("autocmd")
-    "Set UTF-8 as the default encoding for commit messages
+    " Set UTF-8 as the default encoding for commit messages
     autocmd BufReadPre COMMIT_EDITMSG,MERGE_MSG,git-rebase-todo setlocal fileencodings=utf-8
 
-    "Remember the positions in files with some git-specific exceptions"
+    " Remember the positions in files with some git-specific exceptions"
```

Comment:
correct

Reply:
In that case, could I ask to split them out? I was really confused because I am not familiar with the Vim syntax, so I thought you introduced functional changes here that I do not understand.

Reply:
Ah, I see that you explained that part in the commit message. No need to split anything anymore!